back-end/resources/room_config.json

```diff
@@ -253,79 +234,5 @@
       ]
     }
   ],
-  "general_events": [
```

Comment:
Let's keep them in and change them next week when we will implement the front-end buttons

Reply:
Sure, I will write a nice demo configuration